lib/srv/db/sqlserver/connect.go

```diff
@@ -76,6 +76,7 @@ func (c *connector) Connect(ctx context.Context, sessionCtx *common.Session, log
 		LoginOptions: options,
 		Encryption:   msdsn.EncryptionRequired,
 		TLSConfig:    tlsConfig,
+		PacketSize:   loginPacket.PacketSize(),
```

Comment:
Wireshark has TDS packet filter that allows to visualise SQLServer traffic. Since the tsh proxy db has `--tunnel` is pretty easy to look/debug db wire protocol.